lib/cancan/controller_resource.rb

```diff
@@ -138,7 +138,7 @@ def parent_authorization_action
     end
 
     def id_param
-      @params[id_param_key].to_s if @params[id_param_key]
+      @params[id_param_key].to_s if @params[id_param_key].present?
```

Comment:
can you please add a test which fails before your fix and that is fixed now? I also change the base to `feature/v2` branch so we can release this with next major release. Thanks!